python/ray/serve/_private/http_proxy.py

```diff
@@ -7,24 +7,30 @@
 import pickle
 import socket
 import time
-from typing import Dict, List, Optional, Tuple, Any, Type
+import grpc
```

Comment:
nit: let's change the name of this file? proxy.py is good to me.

Reply:
agreed, but not yet. There will be a bunch of name changing in the follow up XP